registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -3392,6 +3392,38 @@
           "tokenManagerType": "mintBurn"
         }
       ]
+    },
+    "0x8a389d95a7c7cbb117524f5d8eaf65a77a089b9e2bfa1e9a16e385bba543b02b": {
+      "tokenId": "0x8a389d95a7c7cbb117524f5d8eaf65a77a089b9e2bfa1e9a16e385bba543b02b",
+      "deployer": "0xdf9574d2B031882ea999220F3F7C87c4Ce10846a",
+      "originalMinter": null,
+      "prettySymbol": "OXYZ",
+      "decimals": 18,
+      "originAxelarChainId": "ethereum",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/oxyz.svg"
```

Comment:
The logo looks really small. Do you have any other size you can update it with? Thanks